### ISSUE TITLE
Add preferred pronoun info to contacts

### ISSUE THREAD

**Issue by timbl** [solid/contacts-pane#26]
This is important for people to be able to declare on their public profile, and for others who have copies of their name be able to access it also at the same time as their name.

Of course using it in software in place of this ubiquitous 'their' singular then becomes possible.

Not sure what ontology has it.  A quick search of LOV for pronoun does not lead to personal preferred  pronouns.

**Comment by timbl** [comment]
https://lov.linkeddata.es/dataset/lov/terms?q=pronoun&page=1

**Comment by TallTed** [comment]
Maybe [GSSO - the Gender, Sex, and Sexual Orientation ontology](https://www.ebi.ac.uk/ols/ontologies/gsso) has something suitable...

**Comment by jeff-zucker** [comment]
From the ontology @TallTed mentioned

 

    <owl:AnnotationProperty rdf:about="https://www.wikidata.org/wiki/Property:P6553">
        <owlready_ontology:python_name xml:lang="en">preferred_pronoun</owlready_ontology:python_name>
        <rdfs:label xml:lang="en">preferred pronoun</rdfs:label>
    </owl:AnnotationProperty>

**Comment by TallTed** [comment]
Worth noting -- I discovered GSSO through [a simple Google search for `ontology "preferred pronoun"`](https://www.google.com/search?q=ontology+%22preferred+pronoun%22). I have never had much luck in finding ontologies through the tools that specialize in ontology indexing (like LOV) -- while generalized indexers (like Google) have generally delivered just what I was looking for. I wonder why that is?

**Comment by jeff-zucker** [comment]
I have also just been discussing this with some people, and apparently there is some objection to the adjective "preferred" and that plain "pronoun" is, um preferred.  Wikipedia says "There exists some disagreement on whether or not to refer to PGPs as "preferred". Some people omit the word "preferred", calling them "gender pronouns" or simply "pronouns" to emphasize that correct use of pronouns is a social obligation rather than an individual preference"  https://en.wikipedia.org/wiki/Preferred_gender_pronoun

**Comment by TallTed** [comment]
@jeff-zucker -- Good point. I know a number of people for whom the pronouns associated with their self-identification tend to be difficult for strangers to guess, and they have generally communicated their "pronouns", not their "preferred pronouns".

Indeed, the [Wikidata page cited above for _preferred pronoun_](https://www.wikidata.org/wiki/Property:P6553) has two "Also known as" entries -- _personal pronoun_ and _pronoun_. I don't think anyone I know would object to either of these labels for this attribute, regardless of their self-identification.

**Comment by jeff-zucker** [comment]
Yes, I agree, I've head "my pronouns are ..." and "what are your pronouns?" many times but never with the word "preferred".
